kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -1021,7 +1021,7 @@ def _init_cterm(
         'MEMORYUSED_CELL': intToken(0),
         'WORDSTACK_CELL': KApply('.WordStack_EVM-TYPES_WordStack'),
         'PC_CELL': intToken(0),
-        'GAS_CELL': intToken(9223372036854775807),
+        'GAS_CELL': KEVM.inf_gas(KVariable('VGAS')),
```

Comment:
With `inf_gas` by default, we could remove the `kevm.infiniteGas()` calls from the test suite.

Reply:
`kevm.infiniteGas()` is removed from tests in https://github.com/runtimeverification/evm-semantics/commit/b4335416a272d175c8a811e79ff0fc7dc430338e.